packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx

```diff
@@ -50,21 +53,34 @@ export const HoldPane = (): JSX.Element => {
     };
   }, [startTime]);
 
+  const resumeSpinner = (): JSX.Element => {
+    return <Spinner label={strings.resumingCallButtonLabel} labelPosition={'right'} />;
+  };
+
   return (
     <Stack styles={paneStyles}>
       <Stack horizontal styles={holdPaneContentStyles}>
         <Text styles={holdPaneTimerStyles}>{elapsedTime}</Text>
         <Text styles={holdPaneLabelStyles}>{strings.holdScreenLabel}</Text>
         <PrimaryButton
-          text={strings.resumeCallButtonLabel}
-          ariaLabel={strings.resumeCallButtonAriaLabel}
+          text={!resumingCall ? strings.resumeCallButtonLabel : undefined}
+          ariaLabel={!resumingCall ? strings.resumeCallButtonAriaLabel : strings.resumingCallButtonAriaLabel}
           styles={resumeButtonStyles}
-          onClick={() => {
-            /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
-            holdButtonProps.onToggleHold();
+          disabled={resumingCall}
+          onClick={async () => {
+            setResumingCall(true);
+            try {
+              /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+              await holdButtonProps.onToggleHold();
+            } catch (e) {
+              setResumingCall(false);
+              throw e;
+            }
           }}
           data-ui-id="hold-page-resume-call-button"
-        />
+        >
+          {resumingCall && resumeSpinner()}
```

Comment:
Is there a reason to choose a function expression instead of JSX syntax?
i.e. `resumeSpinner()` instead of `<ResumeSpinner />`